shared/constants/chat2/convostate.tsx

```diff
@@ -473,8 +473,6 @@ const createSlice: Z.ImmerStateCreator<ConvoState> = (set, get) => {
   }
 
   // used by loadMoreMessages
-  // only let one be in flight at a time
-  let inLoadMore = false
```

Comment:
don't try and do this, it wasn't perfect and we actually rely on some racy behavior when we foreground / get a stale